litellm/proxy/guardrails/guardrail_initializers.py

```diff
@@ -45,6 +45,17 @@ def initialize_lakera(litellm_params, guardrail):
     litellm.logging_callback_manager.add_litellm_callback(_lakera_callback)
 
 
+def initialize_acuvity(litellm_params, guardrail):
+    from litellm.proxy.guardrails.guardrail_hooks.acuvity import AcuvityGuardrail
+
+    _acuvity_callback = AcuvityGuardrail(
+        api_base=litellm_params["api_base"],
+        api_key=litellm_params["api_key"],
+        event_hook=litellm_params["mode"],
+        default_on=litellm_params["default_on"],
```

Comment:
I'm a bit confused about this. If this is how we are instantiating our guardrail class, how do the vendor params end up in our class at all? I'm seeing that they are somehow passed as kwargs, but that's not done here either.

Reply:
everything is done in acuvity class.